pcap-dos.c

```diff
@@ -1295,7 +1295,7 @@ struct device tc90xbc_dev LOCKED_VAR = {
 
 struct device wd_dev LOCKED_VAR = {
               "wd",
-              "Westen Digital",
+              "Western Digital",
```

Comment:
I'm assuming that these fields are descriptive/for human consumption as opposed to being used to match something in data. Obviously changing their values would make output for interpreted things change, but hopefully that's ok.

Reply:
It appears to be used to set the device description field for an interface, so it's for human consumption.  Fixing strings meant for human consumption is OK.